README.md

```diff
@@ -27,6 +27,7 @@ Any LTS version from 8.4.0 (with `--expose-http2`) or 8.8.1 without flags, onwar
 
 * clone the repository
 * `npm i` in the top level directory
+* `npm i -g lerna`
```

Comment:
`lerna` is already installed as a `devDependency`. We probably shouldn't be encouraging people to install it globally as well.